projects/imx6/linux/4.9-wb/linux.arm.conf

```diff
@@ -0,0 +1,6562 @@
+#
```

Comment:
no aarch64 support?

Reply:
I don't shure about imx6, this linux and Aarch64.